### ISSUE TITLE
Channel summary has unexpected colon in end column (sometimes)

### ISSUE THREAD

**Issue by kkappler** [kujaku11/mth5#97]
This issue is related to one I reported in aurora [https://github.com/simpeg/aurora/issues/169](url)

It is happening in the parkfield test dataset  there, but the bogus character seems to appear only after the 
from_runts() method in master_station_run_channel.py method is called (currently around lines 1544-1588).

The error can be reproduced by running in aurora/tests/parkfield/make_parkfield_mth5.py, and then:

```
from mth5.utils.helpers import initialize_mth5
m = initialize_mth5("data/pkd_test_00.h5")
print(m.channel_summary) # works fine
m.channel_summary.to_dataframe() # fails
``` 

which produces the error:
```
dateutil.parser._parser.ParserError: Unknown string format: 2004-09-28T01:59:59.950000+00:
```

**Comment by kujaku11** [comment]
The issue was the time length hard coded into the `channel_summary` data types was set to 30 characters which cut the end time off at the time zone. 

Changed the length to 36, which should be long enough to accommodate a full time string.  Pushed to `master`.

**Comment by kujaku11** [comment]
should be fixed in PR #103
